lib/vrl/tests/src/main.rs

```diff
@@ -21,10 +21,14 @@ use vrl::{
 };
 use vrl_tests::{docs, Test};
 
-#[cfg(not(target_env = "msvc"))]
+#[cfg(all(not(target_env = "msvc"), not(feature = "mimalloc")))]
 #[global_allocator]
 static ALLOC: tikv_jemallocator::Jemalloc = tikv_jemallocator::Jemalloc;
 
+#[cfg(all(not(target_env = "msvc"), feature = "mimalloc"))]
```

Comment:
BTW, if I understand correctly, `target_env = "msvc"` indicates Windows environment, and I think probably mimalloc could be used on Windows since it comes from microsoft (https://github.com/microsoft/mimalloc) and probably could perform well, but here I simply follow the original way to specify it. Let me know if you think otherwise. Thanks.